homeassistant/components/neato/button.py

```diff
@@ -0,0 +1,70 @@
+"""Support for Neato buttons."""
```

Comment:
Needs test or add to `.coveragerc` to exclude

Reply:
Ok, I added the newly added file to the exclude